spec/rspec/matchers/english_phrasing_spec.rb

```diff
@@ -0,0 +1,76 @@
+module RSpec
+  module Matchers
+    RSpec.describe EnglishPhrasing do
+      describe ".split_words" do
+        it "replaces underscores with spaces" do
+          expect(described_class.split_words(:banana_creme_pie)).to \
+            eq("banana creme pie")
+        end
+
+        it "first casts its argument to string" do
+          x = double
+          allow(x).to receive(:to_s).and_return("banana")
+          expect(described_class.split_words(x)).to eq("banana")
+        end
+      end
+
+      describe ".list" do
+        context "given nil" do
+          it "returns value from inspect, and a leading space" do
+            expect(described_class.list(nil)).to eq(" nil")
+          end
+        end
+
+        context "given a Struct" do
+          it "returns value from inspect, and a leading space" do
+            banana = Struct.new("Banana", :flavor)
+            allow(banana).to receive(:inspect).and_return("Delicious")
+            expect(described_class.list(banana)).to eq(" Delicious")
+          end
```

Comment:
`banana` is a struct class, not a struct instance, and it doesn't satisfy the `Struct ===` check of the `list` method:

``` ruby
irb(main):002:0> Person = Struct.new(:name)
=> Person
irb(main):003:0> Struct === Person
=> false
irb(main):004:0> Struct === Person.new
=> true
```

You'd have to do `banana.new` to get an instance that satisfies that check.

It also seems a bit smelly to stub a core method like `inspect`...you could just use `banana_instance.inspect` in the expectation instead.